graphql/tests/projectSettings/queries/project-settings-subscriptions.graphql

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I think you will also need to add data to https://github.com/evergreen-ci/evergreen/blob/main/graphql/tests/task/results.json to complete these tests